mappings/net/minecraft/world/event/listener/GameEventListener.mapping

```diff
@@ -0,0 +1,48 @@
+CLASS net/minecraft/class_59 net/minecraft/world/event/listener/GameEventListener
+	METHOD method_1149 blockUpdate (III)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD method_1150 setBlocksDirty (IIIIII)V
+		ARG 1 minX
+		ARG 2 minY
+		ARG 3 minZ
+		ARG 4 maxX
+		ARG 5 maxY
+		ARG 6 maxZ
+	METHOD method_1151 updateBlockEntity (IIILnet/minecraft/class_55;)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 blockEntity
+	METHOD method_1152 worldEvent (Lnet/minecraft/class_54;IIIII)V
+		ARG 1 player
+		ARG 2 event
+		ARG 3 x
+		ARG 4 y
+		ARG 5 z
+		ARG 6 data
+	METHOD method_1153 addParticle (Ljava/lang/String;DDDDDD)V
+		ARG 1 particle
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+		ARG 8 velocityX
+		ARG 10 velocityY
+		ARG 12 velocityZ
+	METHOD method_1154 playSound (Ljava/lang/String;DDDFF)V
+		ARG 1 sound
+		ARG 2 x
+		ARG 4 y
+		ARG 6 z
+		ARG 8 volume
+		ARG 9 pitch
+	METHOD method_1155 playMusic (Ljava/lang/String;III)V
```

Comment:
This seems incorrect. This would make me think that it would play background music from `/music` or `/newmusic` directories. When it actually plays from `/streaming` because this is the function for playing records I believe.

Reply:
I'll make it playStreaming